### PR TITLE
MAINT: Use copydoc and cleanup some docstring additions

### PR DESCRIPTION
**Pull request by llllllllll**

### CODE REVIEW

**Comment by llllllllll** [comment]
I don't understand the build output at all.

**Comment by cpcloud** [comment]
@llllllllll this should be fixed

**Comment by llllllllll** [comment]
cool, merging on pass

File: blaze/expr/arrays.py [hunk 1]
```diff
@@ -49,11 +53,9 @@ def transpose(expr, axes=None):
         axes = tuple(axes)
     return Transpose(expr, axes)
 
-transpose.__doc__ = Transpose.__doc__
-
-
+@copydoc(Transpose)
 def T(expr):
-    return transpose(expr)
+    transpose
```
Comment (cpcloud) [inline blaze/expr/arrays.py]: does this need to be `return transpose(expr)`?

Comment (llllllllll) [inline blaze/expr/arrays.py]: yes it does

**Comment by cpcloud** [comment]
@llllllllll looks like something strange is going on ... `dot` and `tensordot` are returning incorrect results with this PR

**Comment by llllllllll** [comment]
I accidentally flipped lhs and rhs, I will fix a

**Pull request merged**